app/views/atmosphere/admin/migration_jobs/index.html.haml

```diff
@@ -0,0 +1,25 @@
+%h1
+  = t('migration_jobs.long_title')
+
+%p
+  %table.table.table-hover.table-condensed
+    %tr
+      %th= t('migration_jobs.appliance_type')
+      %th= t('migration_jobs.virtual_machine_template')
+      %th.table-cell-uuid
+        = t('migration_jobs.virtual_machine_template_uuid')
+      %th= t('migration_jobs.compute_site_source')
+      %th= t('migration_jobs.compute_site_destination')
+      %th= t('migration_jobs.status')
+      %th= t('migration_jobs.last_update')
+
+    - @migration_jobs.each do |migration_job|
+      %tr
+        %td= migration_job.appliance_type.name if migration_job.appliance_type
```

Comment:
You can also think about delivering some kind of decorator, which will remove if conditions from the view.